admin_app/next.config.mjs

```diff
@@ -0,0 +1,8 @@
+/* @type {import('next').NextConfig} */
+const nextConfig = {
+  experimental: {
+    missingSuspenseWithCSRBailout: false,
```

Comment:
I somehow missed this in previous PRs. Why are we doing this? And is it possible to fix the root issue?

https://nextjs.org/docs/messages/missing-suspense-with-csr-bailout